h2o-docs/src/product/automl.rst

```diff
@@ -268,58 +293,87 @@ The models are ranked by a default metric based on the problem type (the second
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+| GBM_grid_0_AutoML_20181022_221411_model_1              | 0.7491659 | 0.9424442 |            0.3629487 | 0.4991597 | 0.2491604 |
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+| XRT_0_AutoML_20181022_221411                           | 0.7329677 | 0.6034210 |            0.3656316 | 0.4564392 | 0.2083368 |
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+| DRF_0_AutoML_20181022_221411                           | 0.7329626 | 0.6072326 |            0.3671294 | 0.4564434 | 0.2083406 |
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+| DeepLearning_grid_0_AutoML_20181022_221411_model_1     | 0.7286182 | 0.6095040 |            0.3689510 | 0.4586798 | 0.2103872 |
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+| DeepLearning_0_AutoML_20181022_221411                  | 0.6881761 | 0.6433626 |            0.4173049 | 0.4738028 | 0.2244891 |
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+| GLM_grid_0_AutoML_20181022_221411_model_0              | 0.6853161 | 0.6366259 |            0.3936651 | 0.4717700 | 0.2225669 |
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+| DeepLearning_grid_0_AutoML_20181022_221411_model_0     | 0.6742703 | 0.6786356 |            0.4233795 | 0.4833303 | 0.2336081 |
++--------------------------------------------------------+-----------+-----------+----------------------+-----------+-----------+
+
+
+Experimental Features
 ~~~~~~~~~~~~~~~~~~~~~
 
 XGBoost
 '''''''
 
-AutoML now includes `XGBoost <data-science/xgboost.html>`__ in its selection of algorithms used to train the best model.
-This feature is however currently provided with the following restrictions:
-- XGBoost is used only if it is available globally, especially if it hasn't been explicitly `disabled <data-science/xgboost.html#disabling-xgboost>`__.
-- Even so, XGBoost is disabled by default in AutoML when running H2O-3 in multi-node due to current `limitations <data-science/xgboost.html#limitations>`__.
-- XGBoost can however be enabled experimentally in multi-node by setting the environment variable ``-Dsys.ai.h2o.automl.xgboost.multinode.enabled=true`` for every node of the H2O cloud.
+AutoML now includes `XGBoost <data-science/xgboost.html>`__ GBMs (Gradient Boosting Machines) among it's set of algorithms. This feature is currently provided with the following restrictions:
```

Comment:
small typo: `its set` instead of `it's set`

Reply:
Nice catch.  Can you fix this is the follow-up PR where you edit this for for model id changes? I am AFK and want to merge so that you can start branch from master easily for the model id fix. Thanks!

Reply:
will do